src/pixie_cli/install.sh

```diff
@@ -219,26 +226,13 @@ execute curl -fsSL "$(artifact_url)" -o "${INSTALL_PATH}"/px_new
 execute chmod +x "${INSTALL_PATH}"/px_new
 execute mv "${INSTALL_PATH}"/px_new "${INSTALL_PATH}"/px
```

Comment:
I know this is not necessarily within the scope of this change but do you mind adding the sha256 check? It's also in the github releases. (Checking the .asc signature if the user has GPG installed is better but probably significantly more complex to implement)